pos_global_discount_in_line/models/pos_config.py

```diff
@@ -0,0 +1,9 @@
+from odoo import fields, models
+
+
+class PosConfig(models.Model):
+    _inherit = "pos.config"
+
+    global_discount_in_line = fields.Boolean(
+        string="Add Discount in Line", default=False
```

Comment:
```suggestion
        string="Add Discount in Line"
```

it's already `False` by default

Reply:
It is null by default. At least from the point of database there is difference between null and false.

Reply:
For boolean fields, the ORM will convert all falsy values to `False`: https://github.com/odoo/odoo/blob/939ffdfadd82789d8ad17245f48814270755b042/odoo/fields.py#L1210

No default value is the same as `default=False`, so it's just redundant.
In any case, I'm not blocking on this. Just a suggestion